packages/hash/api/src/storage/index.ts

```diff
@@ -102,7 +102,7 @@ export const setupStorageProviders = (
  * @param storageProvider - the provider we're using for file storage
  * @param cache - a cache to store presigned URLs so we don't needlessly create URLs for every download
  */
-export const setupFileProxyHanldere = (
+export const setupFileProxyHanlder = (
```

Comment:
Wait, what? @thehabbos007

Reply:
Wow I did not see I had 2 typos there

Reply:
😂

Reply:
https://github.com/hashintel/hash/pull/1849